BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -210,6 +210,29 @@ public final class ConvivaAnalytics: NSObject {
         }
     }
 
+    /**
+     Puts the session state in a notMonitored state.
+     */
+    public func pauseTracking() {
+        // AdStart is the preferred way to pause tracking according to conviva.
+        client.adStart(sessionKey,
+                       adStream: .ADSTREAM_SEPARATE,
+                       adPlayer: .ADPLAYER_SEPARATE,
+                       adPosition: .ADPOSITION_PREROLL)
+        client.detachPlayer(sessionKey)
+        logger.debugLog(message: "Tracking paused.")
+    }
+
+    /**
+     Puts the session state from a notMonitored state into the last one tracked.
+     */
+    public func resumeTracking() {
+        // AdEnd is the preferred way to resume tracking according to conviva.
+        client.adEnd(sessionKey)
+        client.attachPlayer(sessionKey, playerStateManager: playerStateManager)
```

Comment:
Shouldn't the player be attached before calling `endAd`?

Reply:
I don't think that this would make any difference. But maybe it would be cleaner.